Pipfile

```diff
@@ -7,6 +7,8 @@ name = "pypi"
 apscheduler = "~=3.10"
 mysql-connector-python = "~=8.0"
 python-dotenv = "~=1.0"
+pytest = "*"
+pytest-cov = "*"
```

Comment:
Looks like we need to set the version on these so that they can be updated in a controlled manner.

Reply:
I've updated these to now specify versions based on the latest releases and what was installed on my machine